apps/dashboard/app/api/webhooks/stripe/route.ts

```diff
@@ -151,14 +151,14 @@ async function alertSlack(
           type: "section",
           text: {
             type: "mrkdwn",
-            text: `:bugeyes: New customer ${name} signed up for a two week trial`,
+            text: `:bugeyes: New customer ${name} signed up`,
```

Comment:
In a sep PR we can get rid of this, it was added because free trials weren't supported but we get them in Cushion and also in Slack by default once we remove trials.

Reply:
@coderabbitai create an issue